### PR TITLE
Add Typst to links.md

### PR DESCRIPTION
**Pull request by NicoWeio**
Yes, I'm a fanboy. But while for the moment, LaTeX should probably remain the first language students learn, I do envision more curious students giving [Typst](https://typst.app/) a try.

### CODE REVIEW

**Comment by chrbeckm** [comment]
I am not really in favor of it, as it is an online tool, and so the integration into the rest of the workflow is hindered by that. It's the same point for me with Overleaf.

**Comment by aknierim** [comment]
> as it is an online tool,

It's not a purely online tool, it can still be compiled locally. But: Your point still stands in that it's integration is hindered by it's own design. First, it is still in an earlier phase of development than LaTeX and second, it's lacking features and flexibility that the latter already offers through various packages. Even the scripting part can be done with lua, when using LuaLaTeX.

**Comment by maxnoe** [comment]
> I am not really in favor of it, as it is an online tool, and so the integration into the rest of the workflow is hindered by that. It's the same point for me with Overleaf.

That's not true, these are separate softwares. There is the typst compiler which is a normal installable and locally runable executable and the online overleaf like editor:

https://github.com/typst/typst vs https://typst.app/

**Comment by maxnoe** [comment]
Installation currently requires docker or building from (rust) source though, so not really beginner friendly

**Comment by NicoWeio** [comment]
> it's lacking features and flexibility that the latter already offers through various packages

This is definitely true, but it's worth noting that Typst does have a [package system](https://typst.app/docs/packages/). For the purpose of a common lab report, all needs should be met already.

> Installation currently requires docker or building from (rust) source though

Fair point. It's in the Arch repos, so I didn't notice, but it [hasn't come much further yet](https://repology.org/project/typst/versions).

> That's not true, these are separate softwares. There is the typst compiler which is a normal installable and locally runable executable and the online overleaf like editor

The online editor makes use of WASM, by the way, so the difference is only in the interface.

**Comment by chrbeckm** [comment]
> That's not true, these are separate softwares. There is the typst compiler which is a normal installable and locally runable executable and the online overleaf like editor:
> 
> https://github.com/typst/typst vs https://typst.app/

Ok, I just saw the online type then

**Comment by chrbeckm** [comment]
For this discussion, we could also include
- Lyx (https://www.lyx.org/)
- decker (https://decker.cs.tu-dortmund.de/)

**Comment by chrbeckm** [comment]
> Installation currently requires docker or building from (rust) source though, so not really beginner friendly

I thought a bit more about this point, I think the links are for students, and others, willingly to dig deeper on their own. So I think this shouldn't be a big argument against including it.

**Review by aknierim — APPROVED** [review]

File: links.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ title: Linksammlung
 - [Symbols, Units, Nomenclature and Fundamental Constants in Physics](https://iupap.org/wp-content/uploads/2021/03/A4.pdf){:target="_blank"}
 - [Der normgerechte Umgang mit Größen, Einheiten und Gleichungen](https://www.physik.tu-darmstadt.de/media/fachbereich_physik/phys_studium/phys_studium_bachelor/phys_studium_bsc_praktika/phys_studium_bsc_praktika_gp/phys_studium_bsc_praktika_gp_regeln/Gr_Einh_Gl.pdf){:target="_blank"}
 - ISO 80000: Regeln für mathematische Typographie in der Wissenschaft (Google hilft…)
-- [Typst](https://typst.app/){:target="_blank"}: Eine junge LaTeX-Alternative mit [Online-Editor](https://typst.app/), Echtzeit-Kollaboration und schnellem Rendering
+- [Typst](https://typst.app/){:target="_blank"}: Eine junge LaTeX-Alternative mit built-in Online-Editor, Echtzeit-Kollaboration und schnellem Rendering
```
Comment (NicoWeio) [inline links.md:86]: Ups, danke!

**Pull request merged**